sources/core/Stride.Core.Reflection/TypeDescriptors/SetDescriptor.cs

```diff
@@ -11,40 +12,50 @@ namespace Stride.Core.Reflection
 {
     public class SetDescriptor : CollectionDescriptor
     {
-        private static readonly List<string> ListOfMembersToRemove = new List<string> { "Comparer", "Capacity" };
-
-        private readonly MethodInfo addMethod;
-        private readonly MethodInfo removeMethod;
-        private readonly MethodInfo clearMethod;
-        private readonly MethodInfo containsMethod;
-        private readonly MethodInfo countMethod;
-        private readonly MethodInfo isReadOnlyMethod;
-
+        private static readonly List<string> ListOfMembersToRemove = ["Comparer", "Capacity"];
+
+        private Action<object, object?> AddMethod;
+        private Action<object, object?> RemoveMethod;
+        private Action<object> ClearMethod;
+        private Func<object, object?,bool> ContainsMethod;
+        private Func<object, int> CountMethod;
+        private Func<object, bool> IsReadOnlyMethod;
```

Comment:
Naming convention, private members start with a lowercase letter, same issue with the other descriptors

Reply:
i did that as old collection descriptor had also delegates with pascal case, but ye i can change them to camel

Reply:
changed and simplified names to be constant accross the classes